tendermint/src/validator.rs

```diff
@@ -77,7 +79,7 @@ impl Set {
 
     /// Get total voting power
     pub fn total_voting_power(&self) -> vote::Power {
-        self.total_voting_power
+        self.total_voting_power.unwrap()
```

Comment:
Is there any other way to deal with this than turning it into a panic bomb?
Perhaps the getter should return `Option<vote::Power>` if this is really meant to become optional.

Reply:
Thanks! Makes sense to me so fixed in eab906d1b1287b42d24e552a10876e9357806b36